src/dll/edbgEcmdDll.C

```diff
@@ -1352,7 +1352,19 @@ uint32_t dllPutScom(const ecmdChipTarget & i_target, uint64_t i_address, const e
 }
 
 uint32_t dllPutScomUnderMask(const ecmdChipTarget & i_target, uint64_t i_address, const ecmdDataBuffer & i_data, const ecmdDataBuffer & i_mask) {
-  return ECMD_FUNCTION_NOT_SUPPORTED;
+  uint32_t rc = ECMD_SUCCESS;
```

Comment:
Please add some more details in the commit message like why we are adding this support? is it a new feature in the eBMC or some fix?